contracts/DCAStrategies/DCAStrategies/DCAStrategiesPositionsHandler.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '../../interfaces/IDCAStrategies.sol';
+
+abstract contract DCAStrategiesPositionsHandler is IDCAStrategiesPositionsHandler {
```

Comment:
Could we add a `TODO` to add something similar to [this](https://github.com/Mean-Finance/dca-v2-core/blob/main/contracts/interfaces/IDCAHub.sol#L243)?